src/components/shared/Extensions/Button/index.tsx

```diff
@@ -0,0 +1 @@
+export { default } from './Button';
```

Comment:
according to doc the file extension should be .ts

Reply:
Fixed